docs/paypal.md

```diff
@@ -0,0 +1,173 @@
+# Pay with PayPal
+
+Accept PayPal payments in your iOS app using the PayPal Payments SDK.
+
+## How it works
+
+This diagram shows how your client, your server, and PayPal interact:
+
+// TODO - Get a diagram of the payment flow similar to [this](https://developer.paypal.com/braintree/docs/start/overview#how-it-works)
+
+## Eligibility
+
+PayPal is available as a payment method to merchants in multiple [countries and currencies](https://developer.paypal.com/docs/checkout/payment-methods/).
+
+The PayPal iOS SDK supports a minimum deployment target of iOS 13+ and requires Xcode 13+ and macOS Big Sur 11.3+.
+iOS apps can be written in Swift 5.1+.
+This SDK can be integrated with CocoaPods or Swift Package Manager.
+
+## How to integrate
+
+- [Custom Integration](#technical-steps---custom-integration)
+
+### Know before you code
+
+You will need to set up authorization to use the PayPal Payments SDK. 
+Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID. 
+Follow the steps in [Enable the SDK](https://developer.paypal.com/sdk/in-app/android/#link-enablethesdk) to set up your account and application for PayPal payments.
+
+You will need a server integration to create an order and capture the funds using [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). 
+For initial setup, the `curl` commands below can be used in place of a server SDK.
+
+### Technical steps - custom integration
+
+#### 1. Add the Payments SDK  to your app
+
+#### Swift Package Manager
+
+To add the PayPal package to your Xcode project, select _File > Swift Packages > Add Package Dependency_ and enter https://github.com/paypal/iOS-SDK as the repository URL. Tick the checkboxes for the specific PayPal libraries you wish to include.
+
+In your app's source code files, use the following import syntax to include PayPal's libraries:
+
+```swift
+import PayPal
+```
+
+#### CocoaPods
+
+Include the PayPal pod in your `Podfile`.
+
+```ruby
+pod 'PayPal'
+```
+
+In your app's source code files, use the following import syntax to include PayPal's libraries:
+
+```swift
+import PayPal
+```
+
+#### 2. Create a PayPal button 
+
+Add a `PayPalButton` to your View:
+
+```swift
+let payPalButton = PayPalButton()
+```
+
+#### 3. Initiate the Payments SDK
+
+Create a `CoreConfig` using your client ID from the PayPal Developer Portal:
+
+```swift
+let config = CoreConfig(clientID: "<CLIENT_ID>", environment: .sandbox)
+```
+
+Configure a return URL using your bundle identifier:
+
+```swift
+let returnUrl = "<BUNDLE_IDENTIFIER>" + "://paypalpay"
```

Comment:
The return URL is still a pain point as it's not used in the way that a return URL would typically be used in iOS. Additionally, it has to be all lowercase which causes merchant issues when integrating as their bundle ID often includes mixed cases.